xtable-core/src/test/java/org/apache/xtable/ITConversionController.java

```diff
@@ -260,6 +272,96 @@ public void testVariousOperations(
     }
   }
 
```

Comment:
Here is the case added for UUID testing when Iceberg -> Delta, the content is the same as testVariousOperations